.github/workflows/ci.yml

```diff
@@ -26,7 +26,7 @@ jobs:
           curl -LO https://github.com/koalaman/shellcheck/releases/download/stable/shellcheck-stable.zip
           mkdir ~/.shellcheck
           unzip shellcheck-stable.zip -d ~/.shellcheck
-          mv ~/.shellcheck/shellcheck-stable.exe ~/.shellcheck/shellcheck.exe
+          mv ~/.shellcheck/shellcheck.exe ~/.shellcheck/shellcheck.exe
```

Comment:
@MarkTiedemann Thank you for fixing this, but is this line still meaningful? (Looks like moving the file to the same path..

Reply:
@kt3k Thanks, you are right! I just fixed the file name, didn't notice the `mv` was just a rename.